site/src/scaffold/styles/layout.scss

```diff
@@ -62,6 +62,12 @@
 	}
 }
 
+.example {
```

Comment:
Do we need this? Should we just add some standard padding to `.example`?

Reply:
Yesterday I was thinking that having some space between each item would be easier on the eyes. I looked at it again today, doesn't look too bad so we can probably live without it. Added some padding to `.example` though.